server/models/registration.js

```diff
@@ -0,0 +1,38 @@
+const Sequelize = require('sequelize')
+const sequelize = require('./connection');
+
+module.exports =  sequelize.define('registration', {
+    id: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    work_experience_id: {
+      type: Sequelize.INTEGER,
+      allowNull: true,
+      references: {
+        model: 'work_experience',
+        key: 'id'
+      }
+    },
+    girl_id: {
+      type: Sequelize.INTEGER,
+      allowNull: true,
+      references: {
+        model: 'girl',
+        key: 'id'
+      }
+    },
+    status: {
+      type: Sequelize.STRING,
+      allowNull: true
+    },
+    registration_date: {
+      type: Sequelize.DATEONLY,
+      allowNull: true
```

Comment:
Thanks for adding these!

I'm just copying over my last suggestion from the other pull request - adding an `explanation` or `response` TEXT field, to store a company's response to a girl's registration.  It might not be necessary - you can decide if it would be helpful or unnecessary.

Reply:
I added it